src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -55,6 +55,11 @@ See the LICENSE file in the project root for more information.
     <ItemGroup>
       <AdditionalNativeLibrary Include="$(IlcPath)/sdk/libSystem.Private.CoreLib.Native.a" />
       <AdditionalNativeLibrary Include="$(IlcPath)/framework/System.Native.a" />
+      <AdditionalNativeLibrary Include="$(IlcPath)/framework/System.IO.Compression.Native.a" />
+      <AdditionalNativeLibrary Include="$(IlcPath)/framework/System.Net.Http.Native.a" />
+      <AdditionalNativeLibrary Include="$(IlcPath)/framework/System.Net.Security.Native.a" />
+      <AdditionalNativeLibrary Include="$(IlcPath)/framework/System.Security.Cryptography.Native.Apple.a" Condition="'$(TargetOS)' == 'OSX'"/>
+      <AdditionalNativeLibrary Include="$(IlcPath)/framework/System.Security.Cryptography.Native.OpenSsl.a" />
```

Comment:
Should this have `Condition="'$(TargetOS)' != 'OSX'"`? I believe OpenSsl crypto and Apple crypto are mutually exclusive.